### ISSUE TITLE
Can you add a GLFW section？

### ISSUE THREAD

**Issue by VarLad** [ocornut/imgui#4001]
@ocornut Can you add a GLFW example to the tutorial？Thanks

**Comment by ocornut** [comment]
There are 4 glfw examples in the repository I don’t know what you are asking for.

**Comment by VarLad** [comment]
@ocornut Really sorry😅
I was talking about this https://github.com/ocornut/imgui/wiki/Image-Loading-and-Displaying-Examples

**Comment by ocornut** [comment]
Loading textures has nothing to do with GLFW it has to do with the rendering tech you use and most of them are covered in that tutorial.

**Comment by VarLad** [comment]
@ocornut 
Oh... but then, what about Vulkan？Can you add that？

**Comment by ocornut** [comment]
It’s a little complicated as there’s no single way to load a texture in Vulkan, its not the purpose of dear imgui to provide those tutorials really.

Not that the Vulkan backend doesn’t currently support texture switch (see #914) so currently you would also need to modify the Vulkan backend to match the data expressing your texture data.

TL;DR; there’s currently no simple answer and you’ll need to be looking at Vulkan tutorials rather than Dear ImGui ones.
